client/src/components/Profile/PasswordForm/PasswordForm.tsx

```diff
@@ -0,0 +1,85 @@
+import { Box, Button, CircularProgress, TextField } from '@material-ui/core';
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import useStyles from './useStyles';
+
+interface Props {
+  handleSubmit: (
+    { password, oldPassword }: { password: string; oldPassword: string },
+    {
+      setStatus,
+      setSubmitting,
+    }: FormikHelpers<{
+      password: string;
+      oldPassword: string;
+    }>,
+  ) => void;
+}
+
+export default function ProfileForm({ handleSubmit }: Props): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Formik
+      initialValues={{
+        password: '',
+        oldPassword: '',
+      }}
+      validationSchema={Yup.object().shape({
+        password: Yup.string()
+          .required('Password is required')
+          .max(100, 'Password is too long')
+          .min(6, 'Password too short'),
+        oldPassword: Yup.string()
+          .required('Password is required')
+          .max(100, 'Password is too long')
+          .min(6, 'Password too short'),
+      })}
+      onSubmit={handleSubmit}
+    >
+      {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
+        <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <TextField
+            id="oldPassword"
+            placeholder="Enter current password"
+            fullWidth
+            margin="normal"
+            InputProps={{
+              classes: { input: classes.inputs },
+              disableUnderline: true,
+            }}
+            name="oldPassword"
+            autoComplete="oldPassword"
+            autoFocus
+            helperText={touched.oldPassword ? errors.oldPassword : ''}
+            error={touched.oldPassword && Boolean(errors.oldPassword)}
+            value={values.oldPassword}
+            onChange={handleChange}
+          />
+          <TextField
+            id="password"
+            placeholder="Enter new password"
+            fullWidth
+            margin="normal"
+            InputProps={{
+              classes: { input: classes.inputs },
+              disableUnderline: true,
+            }}
+            name="password"
+            autoComplete="password"
```

Comment:
```suggestion
            autoComplete="new-password"
```
(see [here](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#preventing_autofilling_with_autocompletenew-password)) 👍🏼